server/routes/index.js

```diff
@@ -3,6 +3,6 @@ import api from './api';
 
 const router = express.Router();
 
-router.use('/api', api);
+router.use('/api/v1', api);
```

Comment:
this specifies that this is version one of the api. So all the endpoints will be prefixed with `api/v1`.